lib/tasks/email_overdue_reminder.rake

```diff
@@ -0,0 +1,18 @@
+desc 'Send email reminder about overdue checkins'
+task email_overdue_reminder: :environment do
+  if AppConfig.first.overdue_checkin_email_active?
+    # get all reservations that ended before today and aren't already checked
+    # in
+    overdue_reservations = Reservation.overdue
+    Rails.logger.info "Found #{overdue_reservations.size} reservations "\
+                      'overdue for checkin. Sending reminder emails...'
+    overdue_reservations.each do |overdue_reservation|
+      UserMailer.reservation_status_update(overdue_reservation).deliver
+    end
+    Rails.logger.info 'Done!'
+  else
+    Rails.logger.info 'Overdue check in emails are not sent by admin. '\
```

Comment:
Let's update this message to match the language we end up using [above](https://github.com/YaleSTC/reservations/pull/1293/files#r38877045).